packages/@lwc/module-resolver/src/__tests__/module-errors.spec.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import path from 'path';
+import { resolveModule } from '../index';
+
+describe('resolve individual module', () => {
+    test('throw when no modules are empty', () => {
+        function run() {
+            const customImporter = path.join(__dirname, 'fixtures/errors/empty/empty.js');
+            const expectedImportee = 'empty';
+
+            resolveModule(expectedImportee, customImporter);
+        }
+
+        expect(run).toThrow(
+            new Error('Unable to resolve empty: No ModuleRecords have been defined')
```

Comment:
@diervo Looking at this test case, the resolver behavior looks strange. I don't think there should be a special case when no module are registered and should throw the standard error that the requested module can't be resolved.

Reply:
I wouldn't call it strange, but sure, we can normalize it.

Reply:
I will address this in my PR since its easier.